example_external/CHANGELOG.rst

```diff
@@ -0,0 +1,68 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package example_external
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.4.0 (2025-01-06)
+------------------
+* Change header install path (`#213 <https://github.com/PickNikRobotics/generate_parameter_library/issues/213>`_)
+* Contributors: Auguste Bourgois
+
+0.3.9 (2024-10-27)
+------------------
+
+0.3.8 (2024-03-27)
+------------------
+
+0.3.7 (2024-01-12)
+------------------
+
+0.3.6 (2023-07-31)
+------------------
+
+0.3.5 (2023-07-28)
+------------------
+
+0.3.4 (2023-07-24)
+------------------
+
+0.3.3 (2023-04-13)
+------------------
+
+0.3.2 (2023-04-12)
+------------------
+
+0.3.1 (2023-02-01)
+------------------
+
+0.3.0 (2022-11-15)
+------------------
+
+0.2.8 (2022-11-03)
+------------------
+
+0.2.7 (2022-10-28)
+------------------
+
+0.2.6 (2022-09-28)
+------------------
+
+0.2.5 (2022-09-20)
+------------------
+
+0.2.4 (2022-08-19)
+------------------
+
+0.2.3 (2022-08-05)
+------------------
+
+0.2.2 (2022-08-03)
+------------------
+
+0.2.1 (2022-08-02)
+------------------
+
+0.2.0 (2022-08-01)
+------------------
+
+0.1.0 (2022-07-27)
+------------------
```

Comment:
Would it work to remove all these other older releases from the file, since technically this package just popped up at 0.4.0?

Reply:
good question, I don't have that much experience with the release process yet. 

for new packages in [ros2_controllers](https://github.com/ros-controls/ros2_controllers/blob/master/mecanum_drive_controller/CHANGELOG.rst) @bmagyar just left it as it is

Reply:
Can't help with that one. The way you get this is when running `catkin_generate_changelog --all` really goes through the history to match everything backwards. Not sure if there is anything on the buildfarm relying on this.

Reply:
Thanks! I think I'm just going to start the history at 0.4.0.